include/render.h

```diff
@@ -93,8 +93,8 @@ typedef struct {
 
 extern Render_t render;
 extern ScalerLineHandler_t RENDER_DrawLine;
-void RENDER_SetSize(Bitu width, Bitu height, unsigned bpp, float fps,
-                    double ratio, bool dblw, bool dblh);
+void RENDER_SetSize(
+        Bitu width, Bitu height, unsigned bpp, double fps, double ratio, bool dblw, bool dblh);
```

Comment:
Uh, oh - our clang formatting rules did this?

Reply:
Cool catch @dreamer ; likely an errant carriage return after applying the formatter. Will get this fixed up!